src/world/zone_settings.h

```diff
@@ -0,0 +1,13 @@
+#ifndef SERVER_ZONE_SETTINGS_H
+#define SERVER_ZONE_SETTINGS_H
+
+#include "../common/cbasetypes.h"
+
+struct zone_settings_t
```

Comment:
No need for a whole new file here, if this struct is only being used in message_server.cpp

Reply:
Removed file